### DISCUSSION TITLE
Open Assistant as an AI Tutor

### DISCUSSION THREAD

**Discussion by akhil-datla** [LAION-AI/Open-Assistant#3473] [Ideas]
# Overview
I researched promising datasets that we can incorporate into Open Assistant that would enable it to become an AI tutor for students. I have created a Google Document with relevant links, summaries, and dataset licensing information. Here is the [link](https://docs.google.com/document/d/1tNKL7bWaxpYZkSyhNdQd0z7ApwsKVMwwY1GthFYa4fs/edit?usp=sharing).

# Instructions

1. Create a Jupyter notebook in notebooks/data-augmentation which will download the data (you can upload it to Hugging Face or similar if it isn't already easily available). 
2. In the notebook, convert the data to a simple Q-A format which we need for training, e.g. JSONL where each line has prompt and response, and write it locally.
3. Make a PR with the notebook (but don't include the downloaded data itself).

# Improvement

For the Chain-of-Thought (CoT) datasets, Huu Nguyen proposed that we generate a question for each step instead of step by reasoning. The question will come from the assistant and the answer for the step will come from the human. The assistant helps the human to solve a problem step by step.

**Comment by huu4ontocord** [comment]
Would you like to try the COT->tutor conversion? @akhil-datla

**Comment by akhil-datla** [comment]
I would like to collaborate with another contributor and learn from them. I am new to this process, but I am eager to learn!
